### PR TITLE
Fix demo content

### PR DESCRIPTION
**Pull request by LucaMele**
Fixes Layout issue in preview and modal and adapt window height to content height

### CODE REVIEW

File: src/components/o-modal/_preview.html [hunk 1]
```diff
@@ -37,36 +42,106 @@ <h2>Prämie berechnen</h2>
...
+        right: 20px;
+
+    }
+
+    .axa-modal-preview .m-modal__content {
+        position: relative;
+
+    }
+
+    .axa-modal-preview h2 {
```
Comment (raphaellueckl) [inline src/components/o-modal/_preview.html]: no globals, right?

File: src/components/o-modal/index.js [hunk 2]
```diff
@@ -71,6 +73,33 @@ class AXAModal extends BaseComponentGlobal {
...
+    if (offsetHeight && vh) {
+      const marginTopBottom = (vh - offsetHeight) / 2;
+
+      if (marginTopBottom > 0) {
+        contentElementStyle.marginTop = `${marginTopBottom}px`;
+      } else {
+        contentElementStyle.marginTop = '0px';
+      }
+    }
+  }
```
Comment (raphaellueckl) [inline src/components/o-modal/index.js]: Why styling with css?

File: src/components/o-modal/index.scss [hunk 3]
```diff
@@ -36,18 +37,19 @@
   @include respond-up(md) {
     margin-top: 20vh;
     width: 80vw;
-    height: 60vh;
+    height: auto;
```
Comment (raphaellueckl) [inline src/components/o-modal/index.scss]: isn't this the default? Doesn't look like you override anything here.

File: src/components/o-modal/index.scss [hunk 4]
```diff
@@ -36,18 +37,19 @@
...
   }
 
   @include respond-up(lg) {
     margin-top: 20vh;
     width: 70vw;
-    height: 60vh;
+    height: auto;
```
Comment (raphaellueckl) [inline src/components/o-modal/index.scss]: isn't this the default? Doesn't look like you override anything here.

File: src/components/o-modal/_preview.html (hunk 1 above)
Comment (LucaMele) [reply src/components/o-modal/_preview.html]: yes right but this is not global. is scoped in axa-preview. And this is copy paste from previous :)

File: src/components/o-modal/index.js (hunk 2 above)
Comment (LucaMele) [reply src/components/o-modal/index.js]: cause is calculated

File: src/components/o-modal/index.scss (hunk 3 above)
Comment (LucaMele) [reply src/components/o-modal/index.scss]: im overriding as i set height 100% on the mobile screen

File: src/components/o-modal/index.scss (hunk 4 above)
Comment (LucaMele) [reply src/components/o-modal/index.scss]: im overriding as i set height 100% on the mobile screen

**Pull request merged**